src/Snackbar/index.tsx

```diff
@@ -0,0 +1,205 @@
+import { ThemeProvider } from '@emotion/react';
+import { FC, ReactNode, useEffect, useMemo, useRef, useState } from 'react';
+import { createPortal } from 'react-dom';
+import theme from 'src/styles/theme';
+import { generateAnchorOrigin, MESSAGE_TYPES, SnackbarCloseReason, SnackbarOrigin } from './Snackbar.util';
+import SnackbarContent from './SnackbarContent';
+import { SnackbarWrapper, StyledSnackbarCloseButton, TRANSITION_CLOSE_TIME } from './snackbarStyles';
+import { ReactComponent as CloseIcon } from 'src/Snackbar/assets/closeIcon.svg';
```

Comment:
Change SVG import path to relative location avoid build issues in consuming applications:
`import { ReactComponent as CloseIcon } from './assets/closeIcon.svg';`

This is related to https://github.com/ToyotaResearchInstitute/lakefront/blob/main/CONTRIBUTING.md#using-svgs